test/e2e/tests/swap-eth.spec.js

```diff
@@ -0,0 +1,80 @@
+const { strict: assert } = require('assert');
+const { withFixtures, regularDelayMs } = require('../helpers');
+
+describe('Swap Eth for another Token', function () {
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: 25000000000000000000,
+      },
+    ],
+  };
+  it('Completes a Swap between Eth and Dai', async function () {
+    await withFixtures(
+      {
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+        await driver.clickElement({
+          xpath:
```

Comment:
Any reason why you're using xpath? Could we use `css` selectors to keep it consistent or even better `data-testid` attributes

Reply:
i can switch to css, but there aren't data-testid attributes for the swaps elements

Reply:
@gholmes86 you can add data-testid wherever you need to for the sake of e2e tests

Reply:
but the changes you have made are fine for now